semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -170,25 +170,15 @@ public static boolean isSqlTypeBackendForOtelFormat(
    * @return backend uri based on otel format
    */
   public static Optional<String> getBackendURIForOtelFormat(Event event) {
-    if (SpanAttributeUtils.containsAttributeKey(event, OTEL_DB_CONNECTION_STRING)
-        && SpanSemanticConventionUtils.isValidUri(SpanAttributeUtils.getStringAttribute(event, OTEL_DB_CONNECTION_STRING))) {
-      return Optional.of(SpanAttributeUtils.getStringAttribute(event, OTEL_DB_CONNECTION_STRING));
-    } else {
-      return SpanSemanticConventionUtils.getURIForOtelFormat(event);
-    }
+    return SpanSemanticConventionUtils.getURIForOtelFormat(event);
```

Comment:
Why do we need two variants of this method? - `getBackendURIForOtelFormat(Event event)` and `getBackendURIForOtelFormat(Map<String, AttributeValue> attributeValueMap)`

If, we still need,
```
public static Optional<String> getBackendURIForOtelFormat(Event event) {
  return getBackendURIForOtelFormat(event.getAtttibuteMap())
}
```

Reply:
I also wanted to make it uniform, but I think we should still rely on SpanAttributeUtils for reading from event. 
In existing place where we already read from Map, this method is useful.